.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: CI
+on: push
+
+jobs:
+  checks:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.17"
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ^14
+
+      - name: linting code
+        run: make lint
+
+      - name: checking license on source code
+        run: make license/check
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

```suggestion
        os: [macos-10.15, ubuntu-20.04, windows-2019]
```

I always have nasty/random surprises with "latest" stuff